lib/bazel/affected_targets.go

```diff
@@ -171,6 +116,68 @@ skipTarget:
 	return changedRules, changedTests, nil
 }
 
+func ParallelQuery(start, end, startOutputBase, endOutputBase string, log logger.Logger) (*QueryResult, error, *QueryResult, error, error) {
```

Comment:
5 retvals seems excessive - maybe return a struct instance?

Bonus - struct instance can contain single error fetch method, which can encapsulate the startQueryErr/endQueryErr logic

Reply:
returning a struct instance, not sure about your second suggestion.